build/includes/website.mk

```diff
@@ -36,6 +36,8 @@ site-server: ensure-build-image
 site-static: ensure-build-image
 	-docker run --rm $(common_mounts) --workdir=$(mount_path)/site $(DOCKER_RUN_ARGS) $(build_tag) rm -r ./public
 	-mkdir $(agones_path)/site/public
+	docker run --rm $(common_mounts) --workdir=$(mount_path)/site $(DOCKER_RUN_ARGS) $(build_tag) \
+		bash -c "npx --version"
```

Comment:
I don't mind if we want to leave this in for future debugging, or delete it 🤷🏻 

Maybe delete to cleanup?

Reply:
I've removed the npx version check.